t/14-uds.cpp

```diff
@@ -56,15 +57,15 @@ TEST_CASE("ping", "[connection]") {
     auto server = ts::make_server({"redis-server", redis_config.filename_});
     ep::wait_port<ep::Kind::TCP>(port);
 
-    auto count = 1000;
+    std::size_t count = 1000;
```

Comment:
It is good to change the name of count variable to something that can convey the purpose of 1000